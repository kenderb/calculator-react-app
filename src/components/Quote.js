import React from 'react';

const Quote = () => (<div>Quote</div>);

export default Quote;
