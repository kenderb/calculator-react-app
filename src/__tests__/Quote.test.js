import { create } from 'react-test-renderer';
import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom';
import Quote from '../components/Quote';

test('renders correctly', () => {
  const tree = create(<Quote />).toJSON();
  expect(tree).toMatchSnapshot();
});

test('displays the heading', () => {
  render(<Quote />);
  const number = screen.getByText(
    'Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. â€“William Paul Thurston',
  );
  expect(number).toBeInTheDocument();
});
