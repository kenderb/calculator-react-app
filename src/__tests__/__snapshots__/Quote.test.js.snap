// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<p
  className="sc-fubCfw fybJDl"
>
  Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. â€“William Paul Thurston
</p>
`;
